doc_k8s/linux/docker/Dockerfile.script

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+echo *_*_*_*_*_* Script allowing the automatic creation of the chosen project via the associated dockerfile *_*_*_*_*_*
+echo *_*_*_*_*_* We must be careful to enter the address of the private register *_*_*_*_*_*
+##
+#### It is necessary to adapt the script according to the address of the private register
```

Comment:
Why not adding a parameter in your options to customize the registry address if needed?